buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -64,17 +64,34 @@ timeout(time: 6, unit: 'HOURS') {
                         sh 'exit 1'
                     }
 
+                    if (fileExists("${WORKSPACE}/.copyright_whitelist")) {
+                        WHITE_LIST = readFile '.copyright_whitelist'
+                        WHITE_LIST = WHITE_LIST.split("\\r?\\n")
+                    } else {
+                        WHITE_LIST = ""
```

Comment:
Initialize `WHITE_LIST` to blank first, then you don't have to have an `else` case.